### DISCUSSION TITLE
Feature Request: Copy of Database to OneDrive, DropBox etc

### DISCUSSION THREAD

**Discussion by hypercube33** [massCodeIO/massCode#381] [Ideas]
It'd be cool to have a secondary "shared" database on SharePoint or other services (that handle the sync for you) so a team can share snips of code between eachother, or between machines. I use a desktop, a work laptop, a surface, and a gaming laptop for dev and a few others on my team would benefit if we can share code snips between eachother without a lot of effort by tossing the file into a sync'd folder.

**Comment by antonreshetov** [comment]
You can do this now. Just specify the location of the snippets according to your cloud storage service. https://masscode.io/documentation/sync.html

**Reply by elPixelador** [reply]
Something we've tried at work, is having 3 developers pointing to a single google drive hosted masscode database. If we restart, we can see changes from other developers. I'm assuming at this point it's still geared towards one developer. An awesome feature would be having each developer's client get notified of changes to the database and allow them to refresh their local view.

**Reply by elPixelador** [reply]
Like a toast or something that said there's been an update refresh to see changes.

**Reply by antonreshetov** [reply]
In its current implementation, this is unlikely to be possible. To solve such tasks, it is necessary for the DB to be on the server and for the application to have a socket connection for the logic.

The philosophy of massCode is to be a free application, and such functionality clearly falls into the category of paid, and what is more annoying for everyone is that it must be a paid subscription.

**Comment by elPixelador** [comment]
That makes sense, thanks Anton!
